apiary.apib

```diff
@@ -1961,6 +1961,129 @@ Lists all available commands (and subcommands) and exits.
 ## sys
 
 # Group dns
+DNS link resolver.
+
+## dns [GET /dns{?arg}]
+Multihashes are hard to remember, but domain names are usually easy to
+remember. To create memorable aliases for multihashes, DNS TXT
+records can point to other DNS links, IPFS objects, IPNS keys, etc.
+This command resolves those links to the referenced object.
+
+The following examples assume this DNS TXT record:
+
+```
+ipfs.io TXT "dnslink=/ipfs/QmRzTuh2Lpuz7Gr39stNr6mTFdqAghsZec1JoUnfySUzcy ..."
```

Comment:
Maybe it'd be useful to use the `dig` command for DNS resolution? It'd give less experienced readers the tools they need in order to replicate what's being said. For example:

```sh
	> dig +short TXT ipfs.io
	dnslink=/ipfs/QmRzTuh2Lpuz7Gr39stNr6mTFdqAghsZec1JoUnfySUzcy
```